analyzer/src/funTest/assets/projects/synthetic/gomod-subpkg-expected-output.yml

```diff
@@ -0,0 +1,142 @@
+---
```

Comment:
> Doesn't the code that gets added in this file then need to be removed from from existing expected results file?

Why do you think so? As neither the test nor the test input is changed, the test output remains identical. So, this expected result file needs to be identical with the other one.

Reply:
> Why do you think so?

Because of the commit message which says "Use a separate expected output for each synthetic project". That sounds to me as if you're splitting one big expected output into multiple smaller ones (one per project).

Reply:
> Because of the commit message which says "Use a separate expected output for each synthetic project". That sounds to me as if you're splitting one big expected output into multiple smaller ones (one per project).

Do you have any suggestion how to prevent this kind of misunderstanding?